Makefile

```diff
@@ -401,6 +407,7 @@ clean:
 	@$(REBAR) -r clean
 	@rm -rf .rebar/
 	@rm -f bin/couchjs
+	@rm -f bin/escriptize
```

Comment:
Shouldn't this be `rm -f bin/weatherreport`?

Reply:
Fixed